lib/faker/vehicle.ex

```diff
@@ -0,0 +1,224 @@
+defmodule Faker.Vehicle do
+  import Faker, only: [localize: 1]
+  alias Faker.Util
+
+  @moduledoc """
+  Functions for generating Vehicle related data
+  """
+
+  @doc """
+  Returns a vehicle body style string
+
+  ## Examples
+      iex> Faker.Vehicle.body_style()
+      "Minivan"
+      iex> Faker.Vehicle.body_style()
+      "Hatchback"
+      iex> Faker.Vehicle.body_style()
+      "Crew Cab Pickup"
+      iex> Faker.Vehicle.body_style()
+      "Regular Cab Pickup"
+  """
+  @spec body_style() :: String.t()
+  localize(:body_style)
+
+  @doc """
+  Returns a vehicle drivetrain string
+
+  ## Examples
+      iex> Faker.Vehicle.drivetrain()
+      "4x2/2-wheel drive"
+      iex> Faker.Vehicle.drivetrain()
+      "4x4/4-wheel drive"
+  """
```

Comment:
As a standard rule, we work with 4 examples. Please double check your examples.

Reply:
I added some more examples :+1: